postgresql/values.yaml

```diff
@@ -1,19 +1,18 @@
 architecture: standalone
 auth:
-  existingSecret: ""  # name of existing secret to use for PostgreSQL credentials. 
+  existingSecret: "sonataflow-psql-postgresql" # name of existing secret to use for PostgreSQL credentials.
   secretKeys:
-    userPasswordKey: password  # name of key in existing secret to use for PostgreSQL credentials. Only used when `existingSecret` is set.
-  database: sonataflow  # database instance used by data index and job service
-  username: postgres  # database user name 
-  password: postgres  # database password. Ignored if existingSecret is set.
+    userPasswordKey: "postgres-password" # name of key in existing secret to use for PostgreSQL credentials. Only used when `existingSecret` is set.
+  database: sonataflow # database instance used by data index and job service
+  username: postgres # database user name
 primary:
   persistence:
     size: 2Gi # the database storage size
   service:
     ports:
-      postgresql: 5432  # service port to access the database
-# For OpenShift-related configuration see:
-# https://github.com/bitnami/charts/blob/main/bitnami/postgresql/README.md#differences-between-bitnami-postgresql-image-and-docker-official-image
+      postgresql: 5432 # service port to access the database
+  # For OpenShift-related configuration see:
```

Comment:
do we still need the commented lines? if we do maybe it would be good to move it to the readme file

Reply:
I'll move it to the DB section in the readme.

Reply:
thanks